<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAddEditEmail</class>
 <widget class="QDialog" name="FormAddEditEmail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Write e-mail message</string>
  </property>
  <layout class="QFormLayout" name="m_layout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="m_layoutAdder">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PlainToolButton" name="m_btnAdder">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="MRichTextEdit" name="m_txtMessage" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText" stdset="0">
      <string>Contents of your e-mail message</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="m_lblSubject">
     <property name="text">
      <string>Subject</string>
     </property>
     <property name="buddy">
      <cstring>m_txtSubject</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="m_txtSubject">
     <property name="placeholderText">
      <string>Title of your message</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlainToolButton</class>
   <extends>QToolButton</extends>
   <header>plaintoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MRichTextEdit</class>
   <extends>QWidget</extends>
   <header>mrichtextedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_btnAdder</tabstop>
  <tabstop>m_txtSubject</tabstop>
  <tabstop>m_txtMessage</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormAddEditEmail</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
